### PR TITLE
[Refactoring] Remove some unused classes

### PR DESCRIPTION
**Pull request by chibenwa**
Found by running some intellij inspections.

I failed to run `unused declaration` with just classes on the all project, so I ran it only in some maven submodules...

### CODE REVIEW

File: protocols/api/src/main/java/org/apache/james/protocols/api/handler/MultiLineHandler.java [hunk 1]
```diff
@@ -1,67 +0,0 @@
...
-import org.apache.james.protocols.api.Response;
-
-/**
- * A special {@link LineHandler} which will "buffer" the received lines till a point and the push them all at
- * one to the {@link #onLines(ProtocolSession, Collection)} method
- * 
- *
- * @param <S>
- */
-public abstract class MultiLineHandler<S extends ProtocolSession> implements LineHandler<S> {
```
Comment (rouazana) [inline protocols/api/src/main/java/org/apache/james/protocols/api/handler/MultiLineHandler.java]: strange, it is used in protocols/imap/src/main/java/org/apache/james/protocols/imap/core/IMAPCommandDispatcher.java , no?

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**